overlord/ifacestate/helpers.go

```diff
@@ -429,6 +442,22 @@ func (m *InterfaceManager) autoConnect(task *state.Task, snapName string, blackl
 	return affectedSnapNames, nil
 }
 
+func getAutoconnectDisabled(st *state.State) (map[string]bool, error) {
+	var conns map[string]bool
+	err := st.Get("autoconnect-disabled", &conns)
+	if err != nil {
+		if err != state.ErrNoState {
+			return nil, err
+		}
+		conns = make(map[string]bool)
+	}
+	return conns, nil
+}
+
+func setAutoConnectDisabled(st *state.State, flags map[string]bool) {
```

Comment:
s/flags/conns/ to have the same naming as in `getAutoConnectDisabled()`

Reply:
Thanks, that's a good suggestion, but I'll actually use 'connFlags' consistently in these new methods since we use 'conns' in many places already for actual connections.